File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int anc=0;
        for(int i=0;i<nums.size();i++){
            nums[anc]=nums[i++];
            while(i<nums.size() && nums[i]==nums[anc]) i++;
            anc++;
            i--;
        }
        nums.resize(anc);
        return anc;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        anc = 0 
        n = len(nums)
        for i in range(n): 
            if i>0 and nums[i]==nums[i-1]: 
                continue 
            else: 
                nums[anc]=nums[i] 
                anc+=1 
        return anc

            